### PR TITLE
CR for 20520 - Velocity filter DDE eyebrows

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: b6890be73d26709e854d1fab40d77a88dd0f4b6d available at: https://gist.github.com/19a356b4f353fe0e200d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4730.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4730.dmg

**Comment by PhilipRosedale** [comment]
Nice improvement!

File: interface/src/devices/DdeFaceTracker.cpp [hunk 1]
```diff
@@ -389,10 +391,20 @@ void DdeFaceTracker::decodePacket(const QByteArray& buffer) {
         }
 
         // Use BrowsU_C to control both brows' up and down
-        _coefficients[_browDownLeftIndex] = -_coefficients[_browUpCenterIndex];
-        _coefficients[_browDownRightIndex] = -_coefficients[_browUpCenterIndex];
-        _coefficients[_browUpLeftIndex] = _coefficients[_browUpCenterIndex];
-        _coefficients[_browUpRightIndex] = _coefficients[_browUpCenterIndex];
+        float browUp = _coefficients[_browUpCenterIndex];
+        if (isFiltering) {
+            const float BROW_VELOCITY_FILTER_STRENGHT = 0.75f;
```
Comment (PhilipRosedale) [inline interface/src/devices/DdeFaceTracker.cpp]: typo:  STRENGTH

**Comment by PhilipRosedale** [comment]
fix typo, ready to merge.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 807eab9fc494535e3ef90f4b03dba49ab0a11db7 available at: https://gist.github.com/19a356b4f353fe0e200d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4730.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4730.dmg

**Pull request merged**